src/commands/txCommands/push.js

```diff
@@ -74,28 +57,37 @@ const masterAndResourceTrns$ = Rx.Observable
 		Object.assign({}, masterTrns, resourceTrns)
 	);
 
-// don't run against master! will delete trn content on transifex
-const branchCheck$ = gitBranch$.do(branchName => {
-	if (branchName === 'master') {
-		console.log(
-			'do not run this script on master. it will kill the master resource on Transifex.'
-		);
-		process.exit(0);
-	}
-});
-
-const pushContent$ = txlib.diffVerbose$(masterAndResourceTrns$,txlib.localTrnsMerged$)
+const pushContent$ = txlib.diffVerbose$(masterAndResourceTrns$, txlib.localTrnsMerged$)
 	.flatMap(pushResource$);
 
 module.exports = {
-	command: 'pushNewContent',
+	command: 'push',
 	description: 'push content to transifex',
+	builder: yarg =>
+		yarg.option({
+			project: {
+				alias: 'p',
+				default: txlib.PROJECT,
+			},
+			all: {
+				alias: 'a',
+				default: false,
+			}
+		}),
```

Comment:
would be good to add a [`.conflicts` call](http://yargs.js.org/docs/#api-conflictsx-y) here to indicate that `project` is mutually exclusive with `all`

```js
.conflicts('project', 'all')
```